### PR TITLE
docs(k8s): how to update the CRI of a node pool

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

**Review by farhakennar — APPROVED** [review]

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+meta:
+  title: How to change the Container Runtime Interface (CRI) of an existing cluster
+  description: This page explains how to change the Container Runtime Interface (CRI) of an existing cluster
+content:
+  h1: How to change the Containter Runtime Interface (CRI) of an existing cluster
```
Comment (tgenaitay) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:6]: Typo: container

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+tags: cluster cni docker crio cri-o
+dates:
+  validation: 2023-01-31
+  posted: 2023-01-31
+categories:
+  - containers
+---
+
+A container runtime is, as one of the lowest layers of Kubernetes, the software that (in addition to other tasks) ensures that containers are started and stopped accordingly. Historically, Kubernetes used Docker as the container runtime.  
+However, with ongoing development, it became clear that it was not entirely suitable. With the rapidly evolving world of container runtimes, it became clear that other solutions might be better suited.
```
Comment (tgenaitay) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:17]: "it became clear" repeated twice in paragraph

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx (hunk 1 above)
Comment (bene2k1) [reply containers/kubernetes/how-to/change-container-runtime-interface.mdx:6]:
```suggestion
  h1: How to change the Container Runtime Interface (CRI) of an existing cluster
```

**Comment by nArnoSNES** [comment]
Documentation mention only Kosmos instead of Kapsule and Kosmos.

A warning toward the use of local storage on the node should be added to prevent data loss. Something like: "If local storage had been used on old nodes, data must be copied/moved before moving pods and deleting nodes".

**Comment by arno-scw** [comment]
Documentation mention only Kosmos instead of Kapsule and Kosmos.

A warning toward the use of local storage on the node should be added to prevent data loss. Something like: "If local storage had been used on old nodes, data must be copied/moved before moving pods and deleting nodes".

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+dates:
+  validation: 2023-01-31
+  posted: 2023-01-31
+categories:
+  - containers
+---
+
+A container runtime is a software that (in addition to other tasks) ensures that containers are started and stopped accordingly. It is one the lowest layers of Kubernetes. Historically, Kubernetes used Docker as a container runtime.
+However, with ongoing development, it became clear that it was not entirely suitable. With the rapidly evolving world of container runtimes, other solutions emerged as better suited.  
+The Container Runtime Interface (CRI) was developed as a mean to open up Kubernetes to other container runtimes. CRI is a plugin within a Kubernetes cluster that allows the `kubelet` to use more than one type of container runtime. It consists of a protocol buffer and a gRPC API, as well as libraries, with additional specifications and tools.
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:18]:
```suggestion
The Container Runtime Interface (CRI) was developed as a means to open up Kubernetes to other container runtimes. CRI is a plugin within a Kubernetes cluster that allows the `kubelet` to use more than one type of container runtime. It consists of a protocol buffer and a gRPC API, as well as libraries, with additional specifications and tools.
```

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+At Scaleway, we focus on Containerd, the industry-leading CRI solution. [Containerd](https://containerd.io/) was initially developed by Docker and donated to the Cloud Native Computing Foundation (CNCF). It uses a subset of the original Docker engine and comes with most of Docker's functionality for running containers and managing storage and images - but lacks many developer-oriented features. It is, therefore, suitable for large-scale use as part of a container orchestrator, such as Kubernetes.
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:19]:
```suggestion
At Scaleway, we favor Containerd, the industry-leading CRI solution. [Containerd](https://containerd.io/) was initially developed by Docker and donated to the Cloud Native Computing Foundation (CNCF). It uses a subset of the original Docker engine and comes with most of Docker's functionality for running containers and managing storage and images - but lacks many developer-oriented features. It is, therefore, suitable for large-scale use as part of a container orchestrator, such as Kubernetes. 
```

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+## How to move your pods to a new pool
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
+2. Click <Icon name="more" /> next to the cluster you want to edit and click **More info** to find detailed information about the cluster.
+3. Click the **Pools** tab to view the node pools associated to your cluster.
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:41]:
```suggestion
3. Click the **Pools** tab to view the node pools associated with your cluster. 
```

**Review by tgenaitay — APPROVED** [review]
👍

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  paragraph: This page explains how to change the Container Runtime Interface (CRI) of an existing node pool
+tags: cluster cni docker crio cri-o
+dates:
+  validation: 2023-01-31
+  posted: 2023-01-31
+categories:
+  - containers
+---
+
+A container runtime is a software that (in addition to other tasks) ensures that containers are started and stopped accordingly. It is one the lowest layers of Kubernetes. Historically, Kubernetes used Docker as a container runtime.
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:16]:
```suggestion
A container runtime is a software that,  in addition to other tasks, ensures that containers are started and stopped accordingly. It is one the lowest layers of Kubernetes. Historically, Kubernetes used Docker as a container runtime.
```

File: containers/kubernetes/how-to/change-container-runtime-interface.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+However, with ongoing development, it became clear that it was not entirely suitable. With the rapidly evolving world of container runtimes, other solutions emerged as better suited.  
+The Container Runtime Interface (CRI) was developed as a means to open up Kubernetes to other container runtimes. CRI is a plugin within a Kubernetes cluster that allows the `kubelet` to use more than one type of container runtime. It consists of a protocol buffer and a gRPC API, as well as libraries, with additional specifications and tools.
+At Scaleway, we favor Containerd, the industry-leading CRI solution. [Containerd](https://containerd.io/) was initially developed by Docker and donated to the Cloud Native Computing Foundation (CNCF). It uses a subset of the original Docker engine and comes with most of Docker's functionality for running containers and managing storage and images - but lacks many developer-oriented features. It is, therefore, suitable for large-scale use as part of a container orchestrator, such as Kubernetes.
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/how-to/change-container-runtime-interface.mdx:19]:
```suggestion
At Scaleway, we favor Containerd, the industry-leading CRI solution. [Containerd](https://containerd.io/) was initially developed by Docker and donated to the Cloud Native Computing Foundation (CNCF). It uses a subset of the original Docker engine and comes with most of Docker's functionalities for running containers and managing storage and images - but lacks many developer-oriented features. It is, therefore, suitable for large-scale use as part of a container orchestrator, such as Kubernetes. 
```

**Review by RoRoJ — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

**Pull request merged**